### ISSUE TITLE
Incorrect unit conversion to 'decade'

### ISSUE THREAD

**Issue by pedroteixeira** [josdejong/mathjs#1083]
in `v4.1.1`:
```
> mathjs.unit('20', 'year').to('decade').format()
'2 year
```
I think the expected output would be `2 decade`, right?

I'm guessing some conflict with the `deca` prefix..

**Comment by harrysarson** [comment]
Confirming this behavour happens in the master branch.
```
> mathjs.unit(20, 'year').to('decade')
Unit {
  units: [ { unit: 
     { name: 'year',
       base: [Object],
       prefixes: [Object],
       value: 315576000,
       offset: 0,
       dimensions: [Array] },
    prefix: { name: '', value: 1, scientific: true },
    power: 1 } ],
  dimensions: [ 0, 0, 1, 0, 0, 0, 0, 0, 0 ],
  value: 631152000,
  fixPrefix: true,
  isUnitListSimplified: true }
```


Also `mathjs.unit('20', 'year').to('decades').format()` (note the `s`) fails with:
```
SyntaxError: Unit "decades" not found.
```

`mathjs.unit('20', 'days').to('decade').format()` -> `'0.0054757015742642025 year'`
`mathjs.unit('20', 'days').to('decades').format()` -> Error

**Comment by ericman314** [comment]
This was a result of the `decade` unit having the wrong value (`'year'`) for its `name` property.

I wrote a quick test that revealed there was one other typo: the unit `watt` had the name `'W'`.

`hertz` has the name `'Hertz'`, but there's an alias that points from `hertz` to `'hertz'` which seems resets the name of the `hertz` unit to `'hertz'`. This seems unnecessary; maybe we can remove it and change the name to `hertz`?

The `name` property is redundant, we could easily generate the `names` from the keys.

**Comment by ericman314** [comment]
Should be resolved via #1084

**Comment by josdejong** [comment]
The fix is merged, will be available in the first next release.

**Comment by josdejong** [comment]
Fixed in v4.1.2
